packaging/osx/sharedframework/resources/en.lproj/eula.rtf

```diff
@@ -59,8 +60,8 @@ BY USING THE SOFTWARE, YOU ACCEPT THESE TERMS. IF YOU DO NOT ACCEPT THEM, DO NOT
 {\pntext\f3\'B7\tab}include Distributable Code in malicious, deceptive or unlawful programs; or\par
 {\pntext\f3\'B7\tab}modify or distribute the source code of any Distributable Code so that any part of it becomes subject to an Excluded License. An Excluded License is one that requires, as a condition of use, modification or distribution, that\par
 
-\pard{\pntext\f3\'B7\tab}{\*\pn\pnlvlblt\pnf3\pnindent360{\pntxtb\'B7}}\nowidctlpar\fi-358\li1792\sb120\sa120 the code be disclosed or distributed in source code form; or\cf1\f2\par
-{\pntext\f3\'B7\tab}\cf0\f0 others have the right to modify it.\cf1\f2\par
+\pard{\pntext\f3\'B7\tab}{\*\pn\pnlvlblt\pnf3\pnindent360{\pntxtb\'B7}}\nowidctlpar\fi-358\li1792\sb120\sa120 the code be disclosed or distributed in source code form; or\cf2\f2\par
+{\pntext\f3\'B7\tab}\cf0\f0 others have the right to modify it.\cf2\f2\par

```

Comment:
@piotrpMSFT correct. However, the question becomes do all have to become ENG now, because it wouldn't do to have diff EULAs per language for a product.